### PR TITLE
VoIP Design Improvements

### PR DESCRIPTION
**Pull request by ismailgulek**

### CODE REVIEW

File: MatrixKit/Controllers/MXKCallViewController.m [hunk 1]
```diff
@@ -1174,7 +1190,7 @@ - (void)updateLocalPreviewLayout
     
     CGPoint previewOrigin = self.localPreviewContainerView.frame.origin;
     
-    if (previewOrigin.x != 20)
+    if (previewOrigin.x != (bounds.size.width - _localPreviewContainerViewWidthConstraint.constant - 20.0))
```
Comment (SBiOSoftWhare) [inline MatrixKit/Controllers/MXKCallViewController.m:1193]: We can probably define a constant for the "20.0" value in this method.

File: MatrixKit/Assets/MatrixKitAssets.bundle/en.lproj/MatrixKit.strings [hunk 2]
```diff
@@ -531,10 +531,9 @@
 // Settings keys 
 
 // call string
-"call_waiting" = "Waiting...";
-"call_connecting" = "Call connecting...";
+"call_connecting" = "Connecting...";
```
Comment (SBiOSoftWhare) [inline MatrixKit/Assets/MatrixKitAssets.bundle/en.lproj/MatrixKit.strings:534]:
```suggestion
"call_connecting" = "Connecting…";
```

File: MatrixKit/Assets/MatrixKitAssets.bundle/en.lproj/MatrixKit.strings [hunk 3]
```diff
@@ -531,10 +531,9 @@
...
+"call_ringing" = "Ringing...";
```
Comment (SBiOSoftWhare) [inline MatrixKit/Assets/MatrixKitAssets.bundle/en.lproj/MatrixKit.strings:535]:
```suggestion
"call_ringing" = "Ringing…";
```

File: MatrixKit/Assets/MatrixKitAssets.bundle/en.lproj/MatrixKit.strings (hunk 2 above)
Comment (ismailgulek) [reply MatrixKit/Assets/MatrixKitAssets.bundle/en.lproj/MatrixKit.strings]: 👍

File: MatrixKit/Assets/MatrixKitAssets.bundle/en.lproj/MatrixKit.strings (hunk 3 above)
Comment (ismailgulek) [reply MatrixKit/Assets/MatrixKitAssets.bundle/en.lproj/MatrixKit.strings]: 👍

File: MatrixKit/Controllers/MXKCallViewController.m (hunk 1 above)
Comment (ismailgulek) [reply MatrixKit/Controllers/MXKCallViewController.m]: Yeah, updated.

**Review by SBiOSoftWhare — APPROVED** [review]

**Pull request merged**